error.go

```diff
@@ -0,0 +1,12 @@
+package kubeadmtokentpr
+
+import (
+	"github.com/juju/errgo"
+)
+
+var tokenRetrievalFailedError = errgo.New("token retrieval failed")
+
+// IsTokenRetrievalFailed asserts tokenRetrievalFailedError
+func IsTokenRetrievalFailed(err error) bool {
+	return errgo.Cause(err) == tokenRetrievalFailedError
+}
```

Comment:
For certs we use `certRetrievalFailedError` name, I would like to be consistent.